integration_tests/CMakeLists.txt

```diff
@@ -112,6 +126,12 @@ macro(RUN)
             target_include_directories(${name} PRIVATE ${CMAKE_SOURCE_DIR})
             set_target_properties(${name} PROPERTIES LINKER_LANGUAGE C)
             target_link_libraries(${name} lpython_rtlib)
+            if (${RUN_ENABLE_CPYTHON})
+                target_link_libraries(${name} Python::Python)
+                if (RUN_EXTRAFILES)
+                    file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/${RUN_EXTRAFILES} DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
With respect to https://github.com/lcompilers/lpython/pull/1863#issuecomment-1565401214, we do not yet support the `-I` option for importing/including the cpython module file. The Python module file can be accessed if it is in the same directory as of the `lpython` generated `--show-c` output file. Currently, to make the integration_tests to work we are copying the CPython module file to the binary directory so that it is present in the same directory as of the `lpython` generated `--show-c` output file.

Reply:
That's fine, thanks!